osbuild/mounts.py

```diff
@@ -126,8 +126,21 @@ def __init__(self, args):
     # pylint: disable=no-self-use
     @abc.abstractmethod
     def translate_options(self, options: Dict) -> List:
-        if options.get("readonly", False):
-            return ["-o", "ro"]
+        opts = []
+        if options.get("readonly"):
+            opts += ["ro"]
```

Comment:
nitpick: I'm curious why not use `opts.append()` instead? (I mean also in all cases below).

Reply:
I don't mind changing it to append.